packages/fxa-auth-server/lib/senders/emails/templates/subscriptionAccountFinishSetup/index.stories.ts

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { subplatStoryWithProps } from '../../storybook-email';
+
+export default {
+  title: 'SubPlat Emails/Templates/subscriptionAccountFinishSetup',
+} as Meta;
+
+const createStory = subplatStoryWithProps(
```

Comment:
I don't think we need one for every single conditional, but we might call this one "Full details" and another story called "missing details" with all potential undefined vars not passed in.